redshift_sqlalchemy/dialect.py

```diff
@@ -12,6 +12,9 @@
 except ImportError:
     pass
 else:
+    from alembic.ddl.base import RenameTable
+    compiles(RenameTable, 'redshift')(postgresql.visit_rename_table)
+
     class RedshiftImpl(postgresql.PostgresqlImpl):
```

Comment:
right so why not subclassing.   well @compiles someday might want to take into account the hierarchy of SQLCompiler subclasses.  not really sure.  redshift is a very odd case.